docs/LAYOUT.md

```diff
@@ -89,11 +87,4 @@ However, it's important to be able to control spacing between components, so EDS
 
 ### Adding or Modifying Layouts or Grids
 
-`Layout` and `Grid` components should contain the minimum number of patterns to build most product layouts and compositions. Adding additional variants should be handled with careful consideration.
-
-1. Validate the need for adding an additional layout/variant.
-2.
```

Comment:
I thought this was really weird, and I don't know if the author had a plan for what #2 would say, so I collapsed this line into the paragraph above.